cluster-service/deploy/helm/templates/azure-operators-managed-identities-config.configmap.yaml

```diff
@@ -6,4 +6,83 @@ metadata:
   namespace: {{ .Release.Namespace }}
 data:
   azure-operators-managed-identities-config.yaml: |
-{{ .Values.azureOperatorsManagedIdentitiesConfig | b64dec | indent 4 }}
+    controlPlaneOperatorsIdentities:
```

Comment:
With this approach we should be able to remove the linting exception I added https://github.com/Azure/ARO-HCP/blob/9c74aae617d4dd2007f1aea0a3a2144177e97f18/.yamllint.yml#L7